src/rules/skip-mfa-during-silent-auth.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @title Skip MFA during silent authentication
```

Comment:
I don't think this name is the right one to use for this. Maybe something like "Require MFA only once per session"?

Reply:
Agreed 👌 Updated it to `Require MFA once per session`.